tests/sentry/rules/conditions/test_event_attribute.py

```diff
@@ -844,3 +844,15 @@ def test_does_not_error_with_none(self):
             data={"match": MatchType.EQUAL, "attribute": "exception.type", "value": "SyntaxError"}
         )
         self.assertPasses(rule, event)
+
+    def test_attr_is_in(self):
+        event = self.get_event()
+        rule = self.get_rule(
+            data={"match": MatchType.IS_IN, "attribute": "platform", "value": "php, python"}
+        )
+        self.assertPasses(rule, event)
```

Comment:
Is this saying the event's platform is php?

Reply:
Ya - this is saying "if the platform attr of the event is in `['php', 'python']`